src/Sweep.cpp

```diff
@@ -961,102 +885,101 @@ void InfectSweep(double t, int run) // added run number as argument in order to
+								if (Places[PlaceType][PlaceNumber].close_start_time > HolidayStart)  Places[PlaceType][PlaceNumber].close_start_time	= (unsigned short) HolidayStart;
+								if (Places[PlaceType][PlaceNumber].close_end_time	< HolidayEnd)	 Places[PlaceType][PlaceNumber].close_end_time		= (unsigned short) HolidayEnd;
+
+								for (int PlaceMember = 0; PlaceMember < Places[PlaceType][PlaceNumber].n; PlaceMember++)
 								{
-									if (Hosts[Places[j][k].members[ci]].absent_start_time > l) Hosts[Places[j][k].members[ci]].absent_start_time = (unsigned short)l;
-									if (Hosts[Places[j][k].members[ci]].absent_stop_time < b) Hosts[Places[j][k].members[ci]].absent_stop_time = (unsigned short)b;
+									if (Hosts[Places[PlaceType][PlaceNumber].members[PlaceMember]].absent_start_time	> HolidayStart	) Hosts[Places[PlaceType][PlaceNumber].members[PlaceMember]].absent_start_time	= (unsigned short) HolidayStart;
+									if (Hosts[Places[PlaceType][PlaceNumber].members[PlaceMember]].absent_stop_time		< HolidayEnd	) Hosts[Places[PlaceType][PlaceNumber].members[PlaceMember]].absent_stop_time	= (unsigned short) HolidayEnd;
 								}
 							}
 						}
 				}
 			}
 		}
 
-#pragma omp parallel for schedule(static,1) default(none) shared(t, P, CellLookup, Hosts, AdUnits, Mcells, StateT, ts)
-	for (int tn = 0; tn < P.NumThreads; tn++)	//// loop over threads
-		for (int b = tn; b < P.NumPopulatedCells; b += P.NumThreads)	//// loop/step over populated cells
+#pragma omp parallel for schedule(static,1) default(none) shared(t, P, CellLookup, Hosts, AdUnits, Mcells, StateT, TimeStepNow)
+	for (int ThreadNum = 0; ThreadNum < P.NumThreads; ThreadNum++)	//// loop over threads
+		for (int CellIndex = ThreadNum; CellIndex < P.NumPopulatedCells; CellIndex += P.NumThreads)	//// loop/step over populated cells
 		{
-			Cell* c = CellLookup[b]; //// find (pointer-to) cell.
-			for (int j = ((int)c->L - 1); j >= 0; j--) //// loop backwards over latently infected people, hence it starts from L - 1 and goes to zero. Runs backwards because of pointer swapping?
-				if (ts >= Hosts[c->latent[j]].latent_time) //// if now after time at which person became infectious (latent_time a slight misnomer).
-					DoIncub(c->latent[j], ts, tn); //// move infected person from latently infected (L) to infectious (I), but not symptomatic
-			//StateT[tn].n_queue[0] = StateT[tn].n_queue[1] = 0;
-			for (int j = c->I - 1; j >= 0; j--) ///// loop backwards over Infectious people. Runs backwards because of pointer swapping?
+			Cell* ThisCell = CellLookup[CellIndex]; //// find (pointer-to) ThisCell.
+			for (int LatentPerson = ((int)ThisCell->L - 1); LatentPerson >= 0; LatentPerson--) //// loop backwards over latently infected people, hence it starts from L - 1 and goes to zero. Runs backwards because of pointer swapping?
+				if (TimeStepNow == Hosts[ThisCell->latent[LatentPerson]].latent_time) //// if now after time at which person became infectious (latent_time a slight misnomer).
```

Comment:
So >= has become == 

Fine as long as TimeStepNow increases by 1 each time - so this func gets called every timestesp, the RHS is effectively constant and integer - all of which I think are true?

I prefer >= even so - it feels like there must have been for some reason in the original...